src/wam2layers/config.py

```diff
@@ -49,16 +80,45 @@ class Config(BaseModel):
 
     """
 
-    region: FilePath
-    """Location where the region mask is stored.
+    tagging_region: Annotated[
+        Union[FilePath, BoundingBox], AfterValidator(validate_region)
+    ]
+    """Subdomain delimiting the source/sink regions for tagged moisture.
+
+    You can either specify a path that contains a netcdf file, or a bounding box
+    of the form [west, south, east, north].
+
+    The bounding box should be inside -180, -80, 180, 80; if west > south, the
+    coordinates will be rolled to retain a continous longitude.
 
     The file should exist.
 
     For example:
 
     .. code-block:: yaml
 
-        region: /data/volume_2/era5_2021/source_region_global.nc
+        tagging_region: /data/volume_2/era5_2021/tagging_region_global.nc
+        tagging_region: [0, 50, 10, 55]
+    """
+
+    tracking_region: Optional[
+        Annotated[BoundingBox, AfterValidator(validate_region)]
+    ] = None
+    """Subdomain delimiting the region considered during tracking.
+
+    This is useful when you have global pre-processed data but you don't need
+    global tracking.
+
+    You can specify a bounding box of the form [west, south, east, north].
+
+    The bounding box should be inside -180, -80, 180, 80; if west > south, the
+    coordinates will be rolled to retain a continous longitude.
+
```

Comment:
```suggestion

If it is set to `null`, then it will use full domain of preprocessed data.

```

Let user know this option from documentation.